include/ibamr/INSHierarchyIntegrator.h

```diff
@@ -223,9 +233,16 @@ class INSHierarchyIntegrator : public IBTK::HierarchyIntegrator
 
     /*!
      * Return a pointer to the source strength variable.
+     *
+     * @deprecated Use getVelocityDivergenceVariable() instead.
      */
     SAMRAI::tbox::Pointer<SAMRAI::hier::Variable<NDIM> > getFluidSourceVariable() const;
 
+    /*!
+     * Return a pointer to the velocity divergence variable.
```

Comment:
With approximate projection methods, you don't actually get the value of the divergence that you specify --- so I think we should try to be careful not to imply that this is "the divergence of the velocity" (which is something you get from post-processing).

This seems too wordy, but maybe something like:
```suggestion
     * Return a pointer to the variable that specifies the divergence of the velocity.
```